src/compositions/useModal.ts

```diff
@@ -0,0 +1,38 @@
+import { useBoolean } from '@prefecthq/vue-compositions'
+import { computed, Ref } from 'vue'
+
+type UseDrawer = {
+  value: Ref<boolean>,
+  isVisble: Ref<boolean>,
```

Comment:
Do you feel strongly about `isVisible` vs `isOpen` which is what `useDrawer` uses? I'd like to be consistent with these.

Reply:
nah, it'll just require more work to update uses downstream (if there are any), but I don't feel strongly so updated